cmd/argoexec/commands/emissary.go

```diff
@@ -209,6 +216,36 @@ func NewEmissaryCommand() *cobra.Command {
 	}
 }
 
+func createCommand(name string, args []string, template *wfv1.Template) (*exec.Cmd, *os.File, *os.File, error) {
+	command := exec.Command(name, args...)
+	command.Env = os.Environ()
+	command.SysProcAttr = &syscall.SysProcAttr{}
+	osspecific.Setpgid(command.SysProcAttr)
+	command.Stdout = os.Stdout
+	command.Stderr = os.Stderr
+
+	var stdout *os.File
+	var stderr *os.File
+	var err error
+
+	// this may not be that important an optimisation, except for very long logs we don't want to capture
+	if includeScriptOutput || template.SaveLogsAsArtifact() {
+		logger.Info("capturing logs")
+		stdout, err = os.Create(varRunArgo + "/ctr/" + containerName + "/stdout")
```

Comment:
This will overwrite existing output. I'm not sure if that's what we want or not. Instead, do we want to append to this file each time?

Reply:
No, wo don't want to overwrite.

Reply:
Ok, since we dont want to overwrite I assume this should not be resolved. I will update.